x-pack/test/security_solution_cypress/cypress/e2e/detection_response/rule_management/coverage_overview/coverage_overview.cy.ts

```diff
@@ -197,6 +197,7 @@ describe('Coverage overview', { tags: ['@ess', '@serverless'] }, () => {
       deletePrebuiltRulesAssets();
       preventPrebuiltRulesPackageInstallation();
       createAndInstallMockedPrebuiltRules(prebuiltRules);
+
```

Comment:
let's undo this change, it's the only change in this file and it doesn't provide much value